Week-3/assignment3_4.js

```diff
@@ -0,0 +1,18 @@
+function createStack() {
+  return {
+    
+    items: [],
+    push(item) {
+      this.items.push(item);
+    },
+    pop() {
+      return this.items.pop();
+    },
+  };
+}
+
+const stack = createStack();
+stack.push(10);
+stack.push(5);
+stack.pop(); 
+console.log(stack.items);
```

Comment:
you can still access items outside of createStack. Please modify the code. This code seems as it is mentioned in assignment.

Reply:
Let me check